websites/A/Anime Sama/metadata.json

```diff
@@ -2,13 +2,13 @@
 	"$schema": "https://schemas.premid.app/metadata/1.11",
 	"apiVersion": 1,
 	"author": {
-		"name": "RisingSunLight",
-		"id": "240521747852558347"
+		"name": "Atom Skully",
+		"id": "671037171611729920"
 	},
 	"contributors": [
 		{
-			"name": "Atom Skully",
-			"id": "671037171611729920"
+			"name": "RisingSunLight",
+			"id": "240521747852558347"
 		}
 	],
```

Comment:
Is there a reason you are swapping the contributor and author field?

Reply:
I think it's best, as this is the second time I've modified/added features to this Presence.
What's more, the original author of the Presence doesn't seem to maintain it anymore, nor the others he created (the last commit he made on one of these Presences seems to date back to March 2023).
And if I come up with any other ideas for adding functionality to this Presence, I'll be happy to add them :)
Let me know what you think and if you find this change legitimate. If not, I'll change it according to your guidelines.